Adds custom banana slip sound

JohnGinnane opened the pull request:
Still have 5% to make the oof sound

:cl: JohnGinnane
soundadd: Banana peels now have their own slip sound
/:cl:

[why]: # More sounds mean more fun

Chnkr commented on code/game/turfs/open.dm:
```diff
@@ -198,7 +198,12 @@
 			if(prob(95))
-				playsound(C.loc, 'sound/misc/slip.ogg', 50, 1, -3)
+				// Hippie Start - custom sounds for banana
+				if (istype(O, /obj/item/grown/bananapeel))
```
Add a `custom_slip_sound` var on objects, if it's set use that, otherwise use slip sound

JohnGinnane replied:
Gonna wait for my other PR to be merged before that, as I'll add it to the /obj/item class as well

JohnGinnane commented:
@HippieStation/head thoughts??

bardofrage commented:
Why not

fluxcapacitor1337 commented:
after a long period of serious thought I have decided to +1 this

Spankmaster1066 commented:
Why isn't this in yet +1

banthebantz commented:
+1

Dudeinyourbackyard commented:
yeah +1 oof

JohnGinnane commented:
I'll work on this before merging to make it more.. accessible ))

JohnGinnane commented:
Alrighty, I've refactored it so that *any* object could have an alternate slip sound, much like my alternate_screams. If there isn't one it'll use the default one. Tested this with water chempile and it still made the noise. Tested with normal soap and soap whose alternate slip sound list I added a sound to and it played.

@Chnkr can you re-review? I also think I have +3 now too

JohnGinnane commented:
@Chnkr or anyone else
